runtime/Java/src/org/antlr/v4/runtime/BufferedTokenStream.java

```diff
@@ -441,15 +441,17 @@ protected int previousTokenOnChannel(int i, int channel) {
 
 	@Override
 	public String getText() {
+		fill();
```

Comment:
The old behavior was correct here. See https://github.com/antlr/antlr4/pull/2699#discussion_r504692562.

Reply:
Agreed. This matches up with how getTokens() method works which is just use the current list of tokens. We should also fix up C# and C++ to do the same?